src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -417,6 +417,31 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         execCond("docker rmi -f ${id}") || true
     }
 
+    def 'check lables are correctly applied to build'() {
+        given:
+        String id = 'id10'
+        temporaryFolder.newFile('Dockerfile') << """
+            FROM alpine:3.2
+        """.stripIndent()
+        buildFile << """
+            plugins {
+                id 'com.palantir.docker'
+            }
+
+            docker {
+                name '${id}'
+                labels(['test_label': 'test_value'])
+            }
+        """.stripIndent()
+        when:
+        BuildResult buildResult = with('docker').build()
+
+        then:
+        buildResult.task(':docker').outcome == TaskOutcome.SUCCESS
+        exec("docker inspect --format '{{.Config.Labels}}' ${id}").contains("test_label")
```

Comment:
Interestingly, this gives me errors when trying to specify `json` in the format, even though I can run the following locally if I don't remove the image after building it:

```
$ docker inspect --format '{{json .Config.Labels}}' id10
{"test_label":"test_value"}
```